assets/bloomberg.yaml

```diff
@@ -0,0 +1,131 @@
+      websites that have financially relevant information. This makes up the majority
+      of FinPile.
+        2. News (38B tokens) - Includes all news sources relevant to the financial
+      community, excluding news articles written by Bloomberg journalists. Overall,
+      there are hundreds of English news sources in FinPile including "Bloomberg
+      Transcripts", which are transcripts of Bloomberg TV news.
+        3. Filings (14B tokens) - Includes financial statements prepared by (public)
+      companies and made available to the general public.  In the dataset, a majority
+      of the filings come from EDGAR, which is the SEC's online database.
+        4. Press (9B tokens) - Includes press releases typically issued by companies
+      that are financially relevant.
+        5. Bloomberg (5B tokens) - Includes Bloomberg authored news and other documents
+      such as opinions and analyses. The largest sources are “Bloomberg News” and
+      “Bloomberg First Word”, the Bloomberg-authored wire of real-time news.
+    explanation: See [[Financial Datasets]](https://arxiv.org/pdf/2303.17564.pdf#subsection.2.1)
+  excluded: ''
+  quality_control: ''
+  # Downstream
+  access:
+    value: closed
+    explanation: See [[Openness]](https://arxiv.org/pdf/2303.17564.pdf#subsection.8.2)
+  intended_uses: Used to train the BloombergGPT model.
+  prohibited_uses: ''
+  monitoring: ''
+  feedback: ''
+
+- type: model
+  name: BloombergGPT
+  # General
+  organization: Bloomberg
+  description: BloombergGPT is a 50 billion parameter large language model that
+    is specifically trained on a wide range of financial data to support a diverse
+    set of natural language processing tasks within the financial industry.
+  created_date:
+    value: 2023-03-30
+    explanation: The date the model was announced in the [[Bloomberg article]](https://www.bloomberg.com/company/press/bloomberggpt-50-billion-parameter-llm-tuned-finance/).
+  url: https://arxiv.org/abs/2303.17564
+  model_card: none
+  modality: text (English)
+  analysis:
+    value: >
+      Authors evaluate the performance of BloombergGPT on two broad categories of
+      tasks, finance-specific and general purpose, on several standard benchmarks.
+      They compare BloombergGPT to the three closest models: GPT-NeoX, OPT-66B and
+      BLOOM-176B. They also report results from the original GPT-3 whenever externally
+      available. They conclude "We achieve strong results on general LLM benchmarks
+      and outperform comparable models on financial tasks. We attribute this, in
+      decreasing order of impact, to 1. a well-curated internal dataset, 2. our
+      unique choice in tokenizer, and 3. an up-to-date architecture."
+    explanation: See [[Evaluation]](https://arxiv.org/pdf/2303.17564.pdf#section.5)
+  size: 50B parameters (dense model)
+  # Construction
+  dependencies: [FinPile, The Pile, C4, Wikipedia]
```

Comment:
They mention this in the paper: "Our model is a decoder-only causal language model based on BLOOM (Scao et al., 2022)." and "In developing BloombergGPT, we benefited from existing code developed as part of the BLOOM effort Scao et al. (2022), showing that a moderately sized team can produce a competitive model on domain-specific data.". 

Should BLOOM be added as a dependency?

Reply:
As I understand, they are using the BLOOM codebase as opposed to the BLOOM model so no, I think what you have is correct.